x-pack/plugins/apm/public/components/alerting/ui_components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -59,7 +60,47 @@ export function AlertDetailsAppSection({
   rule,
   alert,
   timeZone,
+  setAlertSummaryFields,
 }: AlertDetailsAppSectionProps) {
+  useEffect(() => {
+    const alertSummaryFields = [
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.actualValue"
+            defaultMessage="Actual value"
+          />
+        ),
+        value: formatAlertEvaluationValue(
+          alert?.fields[ALERT_RULE_TYPE_ID],
+          alert?.fields[ALERT_EVALUATION_VALUE]
+        ),
+      },
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.expectedValue"
+            defaultMessage="Expected value"
+          />
+        ),
+        value: formatAlertEvaluationValue(
+          alert?.fields[ALERT_RULE_TYPE_ID],
+          alert?.fields[ALERT_EVALUATION_THRESHOLD]
+        ),
+      },
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.source"
+            defaultMessage="Source"
+          />
+        ),
+        value: '-',
+      },
+    ];
+    setAlertSummaryFields(alertSummaryFields);
```

Comment:
I don't fully follow why this component gets this prop from its parent, which is in charge of supplying its props?

Can't `x-pack/plugins/observability/public/pages/alert_details/components/alert_details.tsx` process these fields and pass the processed fields to this component?

Reply:
@CoenWarmer, This is the idea behind this PR. As the PR description mentions, "**For Actionable Observability reviewers**." `alert_details.tsx` is under the Observability plugin and provides the essential and shared fields across rule types. 

However, we need to add more rule-related fields. Here is where the `alert_details_app_section.tsx` kicks in(each rule type will have this component) to choose and format these fields and then send them back to `alert_details.tsx` so they could appear in the same place, aka under AlertSummary.

Reply:
Aren't these fields exported by the APM plugin in a way so you can import them inside the O11y plugin?

Reply:
The fields being passed 'up' are:

```
        value: formatAlertEvaluationValue(
          alert?.fields[ALERT_RULE_TYPE_ID],
          alert?.fields[ALERT_EVALUATION_VALUE]
        ),
```

`formatAlertEvaluationValue` is being imported from Observability. 

And the field values don't seem APM specific: `alert?.fields[ALERT_RULE_TYPE_ID]`, `alert?.fields[ALERT_EVALUATION_VALUE]`. 

I struggle to see why this can't be done from within the Obs app itself?

Reply:
@CoenWarmer, `alert?.fields[ALERT_EVALUATION_VALUE]` is not APM specific, that's right. 
But for the other rule types, we want it to be separate from the AlertSummary. e.g. for the Metric threshold rule, we will have 1 or more metrics, and each one of them has an Evaluation value, and we want to show it alongside its chart. I think you have something similar for the SLO

Reply:
I also suggested a couple of APM specific fields that I think are borderline required for making the alert details page useful for APM users.

Reply:
@sqren, thanks for the feedback. Added 👍🏻